pallets/process-validation/src/restrictions.rs

```diff
@@ -35,6 +36,7 @@ where
     match *restriction {
         Restriction::None => true, // TODO implement some actual restrictions
         Restriction::FixedNumberOfInputs { num_inputs } => return inputs.len() == num_inputs as usize,
+        Restriction::FixedNumberOfOutputs { num_outputs } => return _outputs.len() == num_outputs as usize,
```

Comment:
`_outputs` is now used so should be renamed to `outputs`. Just a rust lang convention